libbeat/publisher/publish.go

```diff
@@ -176,6 +176,12 @@ func (publisher *PublisherType) Init(
 	publisher.disabled = *publishDisabled
 	if publisher.disabled {
 		logp.Info("Dry run mode. All output types except the file based one are disabled.")
+	} else if len(configs) == 0 {
+		logp.Info("No output set. Fall back to standard output elasticsearch on localhost:9200")
```

Comment:
The intention is that the beats have a "reasonable" default behaviour also without a config file. That is to make it even easier to get started. Instead of logp.Info we could use logp.Warn above